TicTacToe/TicTacToe.cs

```diff
@@ -14,15 +14,20 @@ class Program
 
         public static void Main()
         {
+
             do
             {
                 DrawBoard();
                 GetInput();
 
             } while (!CheckForWin() && !CheckForTie());
 
+
+
+
+
             // leave this command at the end so your program does not close automatically
-            Console.ReadLine();
+            //Console.ReadLine();
         }
 
         public static void GetInput()
```

Comment:
This methods does a lot more than just "get input". Try to keep your method names in sync with what they actually do. In this case, I would split this into a `GetInput`, `PlaceMark` (which you already have), `CheckForEndGame`, and `TogglePlayer` methods. Then in your Program.Main method, call them in the right order.